packages/react-rest/README.md

```diff
@@ -40,33 +40,42 @@ react-rest is a model-driven library for interacting with your backend. By defin
 
 By providing react-rest your schemas, it will be able to fetch related data from your JSON:API backend.
 
-![Data with relationships](doc/attachments/relationships.png)
```

Comment:
This was the only image of code; it's more consistent and accessible to type it out

Reply:
👍